packages/graph/hash_graph/lib/graph/src/api/rest/link.rs

```diff
@@ -195,3 +203,26 @@ async fn inactivate_link<P: StorePool + Send>(
 
     Ok(StatusCode::NO_CONTENT)
 }
+
+#[utoipa::path(
+    get,
+    path = "/links",
+    tag = "Link",
+    responses(
+        (status = 200, content_type = "application/json", description = "List of all links", body = [Link]),
+        (status = 500, description = "Store error occurred"),
+    )
+)]
+async fn get_active_links<P: StorePool + Send>(
+    pool: Extension<Arc<P>>,
+    mut body: Option<Json<Expression>>,
+) -> Result<Json<Vec<Link>>, StatusCode> {
+    read_from_store(
+        pool.as_ref(),
+        &body
+            .take()
+            .map_or_else(|| Expression::Literal(Literal::Bool(true)), |json| json.0),
```

Comment:
Is there a reason we are using `take` + `map_or_else` instead of just `map_or_else`? We could get rid of the `mut` for body if we drop the `take` I think :)

(I suppose this applies for the other query endpoints as well)

Reply:
Yeah, I just noticed that as well - I'm currently moving the endpoints to `POST /.../query`. I will fix this in that PR 🙂